diesel_infer_schema/infer_schema_internals/src/data_structures.rs

```diff
@@ -68,7 +68,7 @@ impl ColumnInformation {
         ColumnInformation {
             column_name: column_name.into(),
             type_name: type_name.into(),
-            nullable: nullable,
+            nullable,
```

Comment:
this syntax landed quite a while ago, right? would be weird to bump min required rustc version for something like this

Reply:
Yes, it landed in the same version as macros 1.1. We already use it elsewhere.